packages/manager/apps/veeam-backup/src/pages/dashboard/helpers.ts

```diff
@@ -1,9 +1,28 @@
+import { VeeamBackupOffer } from '@ovh-ux/manager-module-vcd-api';
 import { TServiceConsumption } from '@/type/service-consumption.type';
-import { VEEAM_BACKUP_CONSUMPTION_VCD_VM_CODE } from '@/pages/dashboard/Dashboard.constants';
+
+import {
+  VEEAM_BACKUP_CONSUMPTION_VCD_VM_CODE,
+  VEEAM_BACKUP_CONSUMPTION_PLAN_CODE,
+} from '@/pages/dashboard/Dashboard.constants';
 
 export const filterConsumedVms = (consumptions: TServiceConsumption[]) => {
   return consumptions.filter(
     (consumption) =>
       consumption.planCode === VEEAM_BACKUP_CONSUMPTION_VCD_VM_CODE,
   );
 };
+
+export const getRealOfferConsumption = (
+  offer: VeeamBackupOffer & {
+    protectionPrimaryRegion?: string;
+    protectionReplicatedRegion?: string;
+  },
+  consumptions: TServiceConsumption[],
+) => {
+  const consumption = consumptions?.find(
+    ({ planCode }) =>
+      planCode === VEEAM_BACKUP_CONSUMPTION_PLAN_CODE[offer.name],
+  );
+  return { ...offer, usedSpaceInGB: consumption?.quantity ?? 0 };
+};
```

Comment:
```suggestion
export const getRealOfferConsumption = (
  offerName: string
  },
  consumptions: TServiceConsumption[],
) => {
  const consumption = consumptions?.find(
    ({ planCode }) =>
      planCode === VEEAM_BACKUP_CONSUMPTION_PLAN_CODE[offerName],
  );
  return { usedSpaceInGB: consumption?.quantity ?? 0 };
};
```

Not necessary add more dependencies that necessary

Reply:
I dont know,

 the goal of the function is to override offer (But not only this)

normally the function should be somehting like this 

export const getConsumption = (
  offerName: 'BRONZE' | 'SILVER' | 'GOLD',  (typescript will not like string as type)
  consumptions: TServiceConsumption[],
) => {
  const consumption = consumptions?.find(
    ({ planCode }) =>
      planCode === VEEAM_BACKUP_CONSUMPTION_PLAN_CODE[offerName],
  );
  return consumption?.quantity ?? 0;
};

i dont like  'BRONZE' | 'SILVER' | 'GOLD',  (the day when types of offerName are updated we have to update this code )

The ideal solution is to have a type offerName in veeam-backup.type.d.ts (since this file is generated, i dont have n ideal solution)

![image](https://github.com/user-attachments/assets/9ca8f318-e70d-4aaa-aa48-40a38ddf5e12)